### DISCUSSION TITLE
apiVersion: keda.sh/v1alpha1

### DISCUSSION THREAD

**Discussion by vladt22** [kedacore/keda#6411] [General]
KEDA is CNCF-graduated yet it's API is still v1alpha1. Will it ever be beta? Or even "v1"?

**Comment by vladt22** [comment]
Perhaps I should clarify the purpose of my enquiry: when I propose to my k8s admin to add Keda to production cluster with many systems running on it I could foresee their question "he wants us to add something ***alpha1 ?!?". Would not be product more trustful if it had a more ... solid version of the API?

**Comment by davogen** [comment]
I think I might have the same problem, I just joined a new company, would like to use KEDA but our policy is quite strict.

I haven't found anything relevant at https://keda.sh/support/, I will probably reach try on of the commercially supported solutions listed there. 

Would love to hear KEDA maintainers thoughts on this.

**Comment by vladt22** [comment]
Not even short answer?
